control-plane/plugin/src/lib.rs

```diff
@@ -64,28 +64,33 @@ pub struct CliArgs {
 impl CliArgs {
     /// Initialize tracing (including opentelemetry).
     pub fn init_tracing(&self) -> TracingFlusher {
-        let git_version = option_env!("GIT_VERSION").unwrap_or_else(utils::raw_version_str);
-        let tags =
-            utils::tracing_telemetry::default_tracing_tags(git_version, env!("CARGO_PKG_VERSION"));
+        init_tracing_with_jaeger(self.jaeger.clone())
+    }
+}
 
-        let fmt_layer = match std::env::var("RUST_LOG") {
-            Ok(_) => FmtLayer::Stderr,
-            Err(_) => FmtLayer::None,
-        };
+/// Init tracing with jaeger.
+pub fn init_tracing_with_jaeger(jaeger: Option<String>) -> TracingFlusher {
+    let git_version = option_env!("GIT_VERSION").unwrap_or_else(utils::raw_version_str);
+    let tags =
+        utils::tracing_telemetry::default_tracing_tags(git_version, env!("CARGO_PKG_VERSION"));
 
-        let jaeger = self.jaeger.as_ref();
+    let fmt_layer = match std::env::var("RUST_LOG") {
+        Ok(_) => FmtLayer::Stderr,
+        Err(_) => FmtLayer::None,
```

Comment:
This goes with Ok(), if RUST_LOG is empty, i.e. RUST_LOG="". Is this expected behaviour?

Reply:
Do you mean if someone explictly sets "" or doesnt set this env at all?

Reply:
@tiagolobocastro , As this binary doesnt have logging like our other services. Does FmtLayer variant (None, Json, Pretty) really matter? Does it also define what trace format we see in jaeger?

Reply:
Lets keep the behavior as is for now. We will set Fmt to Stderr if RUST_LOG is set.